apache-cassandra/dashboards/cassandra-overview.libsonnet

```diff
@@ -0,0 +1,1782 @@
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.3.6',
+};
+
+local numberOfNodesPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'max(cassandra_up_endpoint_count{' + matcher + '})',
+      datasource=promDatasource,
+      legendFormat='{{ cluster }}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Number of nodes',
+  description: 'The sum of unique instances being reported.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'green',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.3.6',
+};
+
+local numberOfDownNodesPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'max(cassandra_down_endpoint_count{' + matcher + '})',
+      datasource=promDatasource,
+      legendFormat='{{ cluster }}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Number of down nodes',
+  description: 'Number of down nodes in the cluster',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 1,
+          },
+        ],
+      },
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '9.3.6',
+};
+
+local connectionTimeoutsPanel(matcher) = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'cassandra_connection_timeouts_count{' + matcher + '}',
+      datasource=promDatasource,
+      legendFormat='{{ connection }} - {{ instance }} - {{ cluster }}',
```

Comment:
Not quite sure I agree with this legend format 🤔 

It may just be me, but I feel this could get really overwhelming having both the IP *and* the instance name, as both are quite long strings. 
One option could be to display the legend as a right-aligned table, which would make it a little easier?

@gloriasc What are your thoughts on this? (See below image from Keith for reference "Connection timeouts")
I think I prefer the style below with only `connection` and `cluster`
![image](https://user-images.githubusercontent.com/1282515/220900251-5adefbf1-2811-4adb-b28a-b92f2df5a275.png)

Reply:
I changed it to be just connection and cluster 👍

Reply:
Agreed! One need to be able to identify the instance but between those options, the shorter the better :)